src/Features/LanguageServer/Protocol/Features/Options/WorkspaceConfigurationOptionsStorage.cs

```diff
@@ -16,34 +16,27 @@ public static WorkspaceConfigurationOptions GetWorkspaceConfigurationOptions(thi
                                                globalOptions.GetOption(EnableOpeningSourceGeneratedFilesInWorkspaceFeatureFlag));
 
     public static readonly Option2<StorageDatabase> Database = new(
-        "FeatureManager/Storage", nameof(Database), WorkspaceConfigurationOptions.Default.CacheStorage,
-        new LocalUserProfileStorageLocation(@"Roslyn\Internal\OnOff\Features\Database"));
+        "Storage", "Database", WorkspaceConfigurationOptions.Default.CacheStorage);
```

Comment:
Is this a breaking change?

Reply:
It's not as this option is not public.

Reply:
ok so what happens when the user upgrades VS if the old option was set?

Reply:
Not sure what you mean. The registry key is unchanged. The option will be read from the same registry key. Just the internal name changed here.

Reply:
See in `VisualStudionOptionsStorage`:
```C#
{"Storage_CloudCacheFeatureFlag", new FeatureFlagStorage(@"Roslyn.CloudCache3")},
{"Storage_Database", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "Database")},
```